### PR TITLE
Contact Information text is overlapping with Start Test button

### PR DESCRIPTION
**Pull request by v-hclaveria**
Proposed solution, pending to be tested on the QA environment.
Change on styling, adding missing tag. Applying the same change to password modal.
@claud-io 
@mjkozicki

### CODE REVIEW

**Comment by v-hclaveria** [comment]
image

File: www/templates/account/includes/modals/contact-info.php [hunk 1]
```diff
@@ -31,9 +31,11 @@
             <input type="hidden" name="id" value="<?= $id; ?>" />
             <input type="hidden" name="type" value="contact-info" />
             <input type="hidden" name="csrf_token" value="<?= $csrf_token ?>" />
-            <div class="save-button">
-                <button type="submit" class="pill-button blue">Save</button>
-            </div>
+            <div class="section">
+				<div class="save-button">
```
Comment (claud-io) [inline www/templates/account/includes/modals/contact-info.php:35]: please format

File: www/assets/css/account.css [hunk 2]
```diff
@@ -189,6 +189,20 @@ body.theme-b {
   border: 0;
 }
 
+.contact-info-modal {
+    height: 38rem !important;
```
Comment (claud-io) [inline www/assets/css/account.css:193]: please check with different resolutions and format code

File: www/assets/css/account.css [hunk 3]
```diff
@@ -218,7 +232,7 @@ body.theme-b {
 .modal-open {
   visibility: visible;
   display: block;
-  z-index: 1001;
+  z-index: 1000001;
```
Comment (lbartoli79) [inline www/assets/css/account.css:235]: Look like this is not correct, this value should be greater than **wpt-header** z-index too, but it fails when we have different media width:
- default z-index: https://github.com/catchpoint/WebPageTest/blob/cd6502f48cac364f3b280fbdd7b84579a1538ab7/www/assets/css/wpt-header.css#L38
- z-index with `@media (min-width: 86em)`: https://github.com/catchpoint/WebPageTest/blob/cd6502f48cac364f3b280fbdd7b84579a1538ab7/www/assets/css/wpt-header.css#L419

File: www/assets/css/wpt-header.css [hunk 4]
```diff
@@ -416,7 +416,7 @@ wpt-header .wptheader_nav_menu details[open] summary span:after {
 
 @media (min-width: 86em) {
   wpt-header {
-    z-index: 999999;
+    z-index: 1000;
```
Comment (lbartoli79) [inline www/assets/css/wpt-header.css:419]: This is going to change header z-index everywhere, are you sure it does not break anything?

**Pull request closed without merging**